include/sender_cadmium.hpp

```diff
@@ -105,7 +137,16 @@ class Sender {
             }   
         }
 
-        // external transition
+        /**
+	* Function that performs external transition.
+        * Retrieves the messages: if the number of messages
+	* is more than 1, it asserts that only one message is
+        * expected per time unit. After that it determines
+        * and sets the next state based on the current state.
+        * It also sets the time of next internal transition. 
+	* @param e time variable
+	* @param mbs message bags
+	*/
```

Comment:
The alignment of the comments for these files in the github looks perfect. But when we look at the same after downloading in any text editor, it looks disturbed (i.e., alignment is changed). I'm not sure if that's an issue from my end, but I would suggest to double check by downloading the branch. 
Below is the screenshot from my PC,

![image](https://user-images.githubusercontent.com/44588557/63066169-d0fc8680-bed6-11e9-8b41-3d7f51c24c83.png)

Reply:
Done, there were tabs instead of 4 spaces

Reply:
Thanks for the update. I downloaded the branch again now and I see that they are not aligned (this is for all the header files of cadmium). I remember other teams having the similar issue (where it looks perfect from developer's system but it is misaligned in the github or it looks perfect on the github but it is misaligned when you download to your system). I would suggest to give heads up to professor about it as I see that alignment is good on github (but its not the same when downloaded).

Reply:
Resolved by replacing tabs with 4 spaces.